sqlglot/dialects/bigquery.py

```diff
@@ -91,3 +91,18 @@ class Generator(Generator):
             exp.DataType.Type.BOOLEAN: "BOOL",
             exp.DataType.Type.TEXT: "STRING",
         }
+
+        def union_op(self, expression):
+            return f"UNION{' DISTINCT' if expression.args.get('distinct') else ' ALL'}"
+
+        def except_op(self, expression):
```

Comment:
what is the behavior in other engines? are all engines except/intersect distinct implicit?

Reply:
Looks like [Snowflake](https://docs.snowflake.com/en/sql-reference/operators-query.html) does duplicate elimination for INTERSECT and EXCEPT